schemas/TransferStateReceipt.json

```diff
@@ -24,7 +46,19 @@
           }]
         }
       }
+    },
+    "condition_state": {
+      "description": "Transfer state",
```

Comment:
`condition_state` is a Transfer state, with an additional state `"nonexistent"`.
`"nonexistent"` is for when you fetch the state of a transfer that doesn't exist -- it is needed for atomic mode to get the receipt condition).